cloud/google/machineactuator_test.go

```diff
@@ -197,6 +206,16 @@ func createCluster(t *testing.T, config gceconfigv1.GCEProviderConfig, computeSe
 	}
 }
 
+func createTempFile() (string, error){
+	f, err := ioutil.TempFile("","")
+	if err != nil {
+		return "", err
+	}
+	f.WriteString("")
```

Comment:
io.TempFile returns the file created, is the write of the empty string necessary? From the docs:
```
TempFile creates a new temporary file in the directory dir with a name beginning with prefix, opens the file for reading and writing, and returns the resulting *os.File.
```